<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_base">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_bottom">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_move">
           <attribute name="title">
            <string>Move</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="1">
                 <widget class="QPushButton" name="pushButton_motor_all_stop">
                  <property name="text">
                   <string>Stop All - NC</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="pushButton_motor_all_move">
                  <property name="text">
                   <string>Move All - NC</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>Where All - NC</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="2" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_motor_one_pos">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_motor_one_name">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Simulated Motor 1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_motor_two_name">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Simulated Motor 2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_motor_two_pos">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_motor_two_moveto">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_motor_one_moveto">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QPushButton" name="pushButton_motor_one_movego">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>GO</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QPushButton" name="pushButton_motor_two_movego">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>GO</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QComboBox" name="comboBox_motor_one_movetype">
                <item>
                 <property name="text">
                  <string>Relative</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Absolute</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QComboBox" name="comboBox_motor_two_movetype">
                <item>
                 <property name="text">
                  <string>Relative</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Absolute</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_1">
           <attribute name="title">
            <string>Depreciated</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGraphicsView" name="graphicsView"/>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_example">
                <property name="title">
                 <string>GroupBox</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_1">
                     <property name="text">
                      <string>X Pixel:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QLabel" name="label_pixel_y">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Y Pixel:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_pixel_x">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="4">
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_statusBar">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_14">
             <item row="1" column="9">
              <widget class="QLabel" name="label_system_time">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="8">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_status_label">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Status:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_status_msg">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>012345678901234567 XXX </string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_server_label">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Server Address:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_server_address">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>  host:port </string>
               </property>
              </widget>
             </item>
             <item row="0" column="8">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuDebugging">
     <property name="title">
      <string>Debugging</string>
     </property>
     <addaction name="action_pv_list"/>
     <addaction name="action_var_list"/>
    </widget>
    <addaction name="action_about"/>
    <addaction name="menuDebugging"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_set_server"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuSetup"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_pv_list">
   <property name="text">
    <string>PV List</string>
   </property>
  </action>
  <action name="action_var_list">
   <property name="text">
    <string>VAR List</string>
   </property>
  </action>
  <action name="action_toggle_viewport">
   <property name="text">
    <string>Toggle View Port Frame</string>
   </property>
  </action>
  <action name="action_toggle_label_frames">
   <property name="text">
    <string>Toggle Label Frames</string>
   </property>
  </action>
  <action name="action_toggle_click_regions">
   <property name="text">
    <string>Toggle Click Regions</string>
   </property>
  </action>
  <action name="action_random_lines_draw">
   <property name="text">
    <string>Random Lines Draw</string>
   </property>
  </action>
  <action name="action_random_lines_erase">
   <property name="text">
    <string>Random Lines Erase</string>
   </property>
  </action>
  <action name="action_beam_render_test">
   <property name="text">
    <string>Beam Render Test</string>
   </property>
  </action>
  <action name="action_beam_erase">
   <property name="text">
    <string>Beam Erase</string>
   </property>
  </action>
  <action name="action_monitor_callbacks">
   <property name="text">
    <string>Monitor Callbacks</string>
   </property>
  </action>
  <action name="action_config">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="action_camera_positioner_test">
   <property name="text">
    <string>Camera Positioner Test</string>
   </property>
  </action>
  <action name="action_camera_positioner_erase">
   <property name="text">
    <string>Camera Positioner Erase</string>
   </property>
  </action>
  <action name="action_camera_positioner_ping">
   <property name="text">
    <string>Camera Positioner Ping</string>
   </property>
  </action>
  <action name="action_beam_ping">
   <property name="text">
    <string>Beam Ping</string>
   </property>
  </action>
  <action name="action_optics_table_ping">
   <property name="text">
    <string>Optics Table Ping</string>
   </property>
  </action>
  <action name="action_set_server">
   <property name="text">
    <string>Set Server</string>
   </property>
  </action>
  <action name="action_set_motor">
   <property name="text">
    <string>Motor Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
